### PR TITLE
[Qt] Improve Qt QML app example

### PR DESCRIPTION
**Pull request by tmpsantos**
- Add slider for controlling pitch.
- Rounded corners on the map.
- Add slider to flip the map to a satellite (in fact we have two maps in sync).

### CODE REVIEW

**Comment by tmpsantos** [comment]
:eyes: @brunoabinader

File: platform/qt/include/qquickmapboxgl.hpp [hunk 1]
```diff
@@ -83,6 +87,7 @@ class Q_DECL_EXPORT QQuickMapboxGL : public QQuickFramebufferObject
         StyleNeedsSync   = 0x04,
         PanNeedsSync     = 0x08,
         BearingNeedsSync = 0x16,
+        PitchNeedsSync   = 0x32,
```
Comment (brunoabinader) [inline platform/qt/include/qquickmapboxgl.hpp]: Can we use `1 << {2,3,4...}` notation instead? I think it is easier to maintain than using raw values.

File: platform/qt/qmlapp/main.qml [hunk 2]
```diff
@@ -1,35 +1,220 @@
...
+import QtQuick 2.0
+import QtQuick.Controls 1.0
+import QtQuick.Layouts 1.0
 
 import QQuickMapboxGL 1.0
 
 ApplicationWindow {
-    id: window
-    title: qsTr("Mapbox GL QML example")
+    title: "Mapbox GL QML example"
```
Comment (brunoabinader) [inline platform/qt/qmlapp/main.qml]: Can we keep the `qsTr`? Who knows we'll need this translated some day :-)

**Comment by brunoabinader** [comment]
Minor comments, but overall 👍 this is looking awesome!

File: platform/qt/qmlapp/main.qml (hunk 2 above)
Comment (tmpsantos) [inline platform/qt/qmlapp/main.qml]: Does it make sense to have it without a translation file?

**Pull request merged**